src/TestSuite/Fixture/TruncationStrategy.php

```diff
@@ -96,14 +98,13 @@ public function beforeTest(string $test): void
             if (!($db instanceof Connection)) {
                 continue;
             }
+            $schema = $db->getSchemaCollection();
+            $tables = $schema->listTables();
+            $tables = array_intersect($schema->listTables(), $fixtures->lastInserted());
```

Comment:
I'm not in love with the `lastInserted()` approach and would appreciate any ideas on a cleaner solution.

The problem I was aiming to solve was that truncating the 54 fixture tables we have was really slow. The current fixture system skips some of this work by only truncating tables that were used in the current test.

Reply:
> The current fixture system skips some of this work by only truncating tables that were used in the current test.

Our internal fixtures don't have foreign keys setup most of the time. Do we want to support fixtures with foreign keys and truncating subsets of the tables?